src/modules/stores/innovation/innovation.models.ts

```diff
@@ -44,6 +44,7 @@ export type sectionType = {
   section: InnovationSectionEnum;
   status: keyof typeof INNOVATION_SECTION_STATUS;
   actionStatus: keyof typeof INNOVATION_SECTION_ACTION_STATUS;
+  actionCount: number;
```

Comment:
I believe that this change produces undesirable effects (in terms of typing's).

Call me to discuss this please...

Reply:
Fixed DTO